src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &__name {
+    font-size: 1.2em;
+    margin-top: 40px;
+    line-height: 18px;
+    font-weight: $secondary-weight;
+  }
+
+  &__code {
+    font-size: 1em;
+    color: $secondary-color;
+    margin-top: 4px;
+    line-height: 14px;
+  }
+
+  &__stars,
+  &__cost {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__cost {
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-size: 1.2em;
+    color: $secondary-color;
+    line-height: $line;
+    font-weight: $main-weight;
+    font-weight: 400;
```

Comment:
The `font-weight` property is defined twice for the `&__price` class. Consider removing one of these declarations to avoid redundancy and potential conflicts.